node/ksmcc/genesis.json

```diff
@@ -1,17 +1,7 @@
 {
   "name": "ksmcc",
   "id": "ksmcc",
-  "bootNodes": [
-    "/dns4/p2p.cc3-0.kusama.network/tcp/30100/p2p/12D3KooWDgtynm4S9M3m6ZZhXYu2RrWKdvkCSScc25xKDVSg1Sjd",
-    "/dns4/p2p.cc3-1.kusama.network/tcp/30100/p2p/12D3KooWNpGriWPmf621Lza9UWU9eLLBdCFaErf6d4HSK7Bcqnv4",
-    "/dns4/p2p.cc3-2.kusama.network/tcp/30100/p2p/12D3KooWLmLiB4AenmN2g2mHbhNXbUcNiGi99sAkSk1kAQedp8uE",
-    "/dns4/p2p.cc3-3.kusama.network/tcp/30100/p2p/12D3KooWEGHw84b4hfvXEfyq4XWEmWCbRGuHMHQMpby4BAtZ4xJf",
-    "/dns4/p2p.cc3-4.kusama.network/tcp/30100/p2p/12D3KooWF9KDPRMN8WpeyXhEeURZGP8Dmo7go1tDqi7hTYpxV9uW",
-    "/dns4/p2p.cc3-5.kusama.network/tcp/30100/p2p/12D3KooWDiwMeqzvgWNreS9sV1HW3pZv1PA7QGA7HUCo7FzN5gcA",
-    "/dns4/kusama-bootnode-0.paritytech.net/tcp/30333/p2p/12D3KooWSueCPH3puP2PcvqPJdNaDNF3jMZjtJtDiSy35pWrbt5h",
-    "/dns4/kusama-bootnode-0.paritytech.net/tcp/30334/ws/p2p/12D3KooWSueCPH3puP2PcvqPJdNaDNF3jMZjtJtDiSy35pWrbt5h",
-    "/dns4/kusama-bootnode-1.paritytech.net/tcp/30333/p2p/12D3KooWQKqane1SqWJNWMQkbia9qiMWXkcHtAdfW5eVF8hbwEDw"
-  ],
+  "bootNodes": [],
```

Comment:
no more bootnodes?

Reply:
They need to match the bootnodes in the config.toml file. I can add them there or delete them here but since we are not using this at the moment and it will likely need to be updated, I just deleted them from here. I can do add them to the config.toml file if preferred.

Reply:
why do they need to match? that seems off

Reply:
This is a feature not a bug. : )

We are testing the export and load config methods alongside the node implementations by actually exporting and loading the gssmr and ksmcc configuration files, so them needing to be the same means the tests are working.

Reply:
which tests do you mean? in the future there will need to be bootnodes here and we shouldn't require that users copy and paste bootnodes from the genesis into the config, it should be fine to have them only in genesis

Reply:
It is fine to only have them in genesis, it just warns them that the values do not match with the values in the toml config.

Testing the export command and the export method in the dot package both export the values, so if we overwrite the values from the genesis file when exporting, we will have mismatching config files and the continuous integration tests will fail because files have been updated.

Reply:
okay, could you update the tests to get around that? or use a temp genesis file in the tests? we shouldn't need to change the actual genesis files to get tests to work

Reply:
Ok, I reverted the changes to the bootnodes in the ksmcc file. The warning logs will now be displayed again when running `gossamer --node ksmcc` and when running tests because the genesis file and configuration file do not match. I think this is your request? As mentioned in the first comment of this thread, I can also clean up the warning logs by adding the bootnodes to the configuration file but I have reverted the clean up as requested.

Reply:
that seems reasonable, I think you can leave it for now, we don't need to duplicate bootnodes in genesis and config